setup.py

```diff
@@ -51,6 +51,7 @@
             'appnexus = flake8_import_order.styles:AppNexus',
             'edited = flake8_import_order.styles:Edited',
             'pycharm = flake8_import_order.styles:PyCharm',
+            'fromsfirst = flake8_import_order.styles:FromsFirst',
```

Comment:
I believe this could be distributed as a separate style-plugin to flake8-import-order. It's unclear to me whether the maintainers want to maintain another style on top of what they already maintain.

Reply:
Yeah, but I had hoped to not have to have yet another thing in my github. My preference would be to land this here, if that’s acceptable.

Reply:
I wouldn't characterize it quite that way. I think there are benefits for it living in the main repo -- it's more obvious to users that there are other styles, and simpler for them to use if they want this style.

That said, ultimately this is the maintainers' call. I won't be upset if they say no.